utils/auto_reg.py

```diff
@@ -3,6 +3,8 @@
 import random
 import requests
 
+IP = '54.90.55.227'
```

Comment:
This value should be parameterized as we saw in class, for example, add the flag `--ip` so the command to run this could be:
```bash
python utils/auto_reg.py --ip <HOST_IP>
```

Reply:
@calebespinoza  Now is parametrized and use the command:
     `python utils/auto_reg.py --ip <HOST_IP>`